tutorials/iot/ubuntu-kiosk/x11-kiosk.md

```diff
@@ -338,17 +331,22 @@ apps:
 parts:
```

Comment:
as before, missing `base: core18` and `version:` as a string? There's also one more untouched *snapcraft.yaml* on step 9, *Confining Xwayland` missing the same updates.

Reply:
Done